Client/Frontend/Home/HistoryPanel.swift

```diff
@@ -395,6 +395,16 @@ class HistoryPanel: SiteTableViewController, HomePanel {
         }
         return cell
     }
+    
+    func configureDisclosureIndicator(for cell: UITableViewCell) {
+        let disclosureImage = UIImage.templateImageNamed("menu-Disclosure")
+        let disclosureView = UIImageView(image: disclosureImage)
+        let enableColor = UIColor.theme.homePanel.historyEnableDisclosureIndicator
+        let disableColor = UIColor.theme.homePanel.historyDisabelDisclosureIndicator
+        let color = cell.isUserInteractionEnabled ? enableColor: disableColor
+        disclosureView.image = disclosureView.image?.tinted(withColor: color)
+        cell.accessoryView = disclosureView
+    }
```

Comment:
The built-in table view disclosure indicator is preferred, as it will be consistent across all the table views in the product that use a disclosure type of cell. The "menu-Disclosure" is for disclosure in popup menus that use stack views.